web/components/atoms/SearchInput.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import "react-dropdown/style.css";
+import ErrorText from "./ErrorText";
+import Select from "react-select";
+
+interface Option {
+  label: string;
+  value: any;
+}
+
+interface Props {
+  options: Option[];
+  label: string;
+  placeholder?: string;
+  error?: string;
+  disabled?: boolean;
+  onChange?: (selectedOption: Option | null) => void;
+}
+
+const SearchableDropdown: React.FC<Props> = ({
+  options,
+  label,
+  placeholder,
+  error,
+  disabled,
+  onChange,
+}) => {
+  const handleChange = (selectedOption: Option | null) => {
+    if (onChange) {
+      onChange(selectedOption);
+    }
+  };
+
+  return (
+    <div className="flex flex-col">
+      <label className="mb-2">{label}</label>
+      
+      <Select
+        options={options}
+        placeholder={placeholder}
```

Comment:
Let me know if this is difficult to do since we're using a third-party components, but would you be able to match the color of the placeholder text to the placeholder text in other inputs? Currently it looks like a Caregiver is selected when only the placeholder is there